### PR TITLE
docs: update MAINTAIN.md

### PR DESCRIPTION
**Pull request by LiteSun**
### What this PR does / why we need it:
 update MAINTAIN.md for better understand

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by tokers — APPROVED** [review]

File: MAINTAIN.md [hunk 1]
```diff
@@ -42,12 +44,14 @@ via `VERSION=x.y.z make release-src`
 2. Package a vote artifact to Apache's dev-apisix repo. The artifact can be created
 via `VERSION=x.y.z make release-src`
 3. Send the [vote email](https://lists.apache.org/thread/q8zq276o20r5r9qjkg074nfzb77xwry9) to dev@apisix.apache.org
+   > After executing the `VERSION=x.y.z make release-src` command, the content of the vote email will be automatically generated in the `/release `directory named `apache-apisix-${x.y.z}-vote-contents`
```
Comment (leslie-tsang) [inline MAINTAIN.md:47]:
```suggestion
   > After executing the `VERSION=x.y.z make release-src` command, the content of the vote email will be automatically generated in the `/release` directory named `apache-apisix-${x.y.z}-vote-contents`
```

File: MAINTAIN.md [hunk 2]
```diff
@@ -27,13 +27,15 @@
 4. Package a vote artifact to Apache's dev-apisix repo. The artifact can be created
 via `VERSION=x.y.z make release-src`
 5. Send the [vote email](https://lists.apache.org/thread/vq4qtwqro5zowpdqhx51oznbjy87w9d0) to dev@apisix.apache.org
+   > After executing the `VERSION=x.y.z make release-src` command, the content of the vote email will be automatically generated in the `/release `directory named `apache-apisix-${x.y.z}-vote-contents`
```
Comment (leslie-tsang) [inline MAINTAIN.md:30]:
```suggestion
   > After executing the `VERSION=x.y.z make release-src` command, the content of the vote email will be automatically generated in the `/release` directory named `apache-apisix-${x.y.z}-vote-contents`
```

**Review by leslie-tsang — CHANGES_REQUESTED** [review]

File: MAINTAIN.md (hunk 1 above)
Comment (leslie-tsang) [reply MAINTAIN.md:47]: Hello there, Docs Lint check failed with `Spaces inside code span elements`

File: MAINTAIN.md (hunk 2 above)
Comment (leslie-tsang) [reply MAINTAIN.md:30]: Ditto

File: MAINTAIN.md [hunk 3]
```diff
@@ -27,13 +27,15 @@
 4. Package a vote artifact to Apache's dev-apisix repo. The artifact can be created
 via `VERSION=x.y.z make release-src`
 5. Send the [vote email](https://lists.apache.org/thread/vq4qtwqro5zowpdqhx51oznbjy87w9d0) to dev@apisix.apache.org
+   > After executing the `VERSION=x.y.z make release-src` command, the content of the vote email will be automatically generated in the `/release` directory named `apache-apisix-${x.y.z}-vote-contents`
```
Comment (spacewander) [inline MAINTAIN.md:30]:
```suggestion
   > After executing the `VERSION=x.y.z make release-src` command, the content of the vote email will be automatically generated in the `./release` directory named `apache-apisix-${x.y.z}-vote-contents`
```

File: MAINTAIN.md [hunk 4]
```diff
@@ -27,13 +27,15 @@
...
 6. When the vote is passed, send the [vote result email](https://lists.apache.org/thread/k2frnvj4zj9oynsbr7h7nd6n6m3q5p89) to dev@apisix.apache.org
 7. Move the vote artifact to Apache's apisix repo
 8. Register the release info in https://reporter.apache.org/addrelease.html?apisix
 9. Create a [GitHub release](https://github.com/apache/apisix/releases/tag/2.10.2) from the minor branch
 10. Update [APISIX's website](https://github.com/apache/apisix-website/commit/f9104bdca50015722ab6e3714bbcd2d17e5c5bb3)
 11. Update APISIX rpm package
-12. Update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601), and create new branch form master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10.2`
+    > Go to [apisix-build-tools](https://github.com/api7/apisix-build-tools) repository and create a new tag named `apisix-${x.y.z}` to automatically generate rpm package, which can be downloaded from artifacts in action
```
Comment (spacewander) [inline MAINTAIN.md:37]:
```suggestion
    > Go to [apisix-build-tools](https://github.com/api7/apisix-build-tools) repository and create a new tag named `apisix-${x.y.z}` to automatically submit the rpm package to yum repo
```

File: MAINTAIN.md [hunk 5]
```diff
@@ -27,13 +27,15 @@
...
+12. Update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601) in [APISIX docker repository](https://github.com/apache/apisix-docker), and create new branch form master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10**.2***`
```
Comment (spacewander) [inline MAINTAIN.md:38]:
```suggestion
12. Update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601) in [APISIX docker repository](https://github.com/apache/apisix-docker), and create new branch form master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10.2`
```

File: MAINTAIN.md [hunk 6]
```diff
@@ -42,12 +44,14 @@ via `VERSION=x.y.z make release-src`
 2. Package a vote artifact to Apache's dev-apisix repo. The artifact can be created
 via `VERSION=x.y.z make release-src`
 3. Send the [vote email](https://lists.apache.org/thread/q8zq276o20r5r9qjkg074nfzb77xwry9) to dev@apisix.apache.org
+   > After executing the `VERSION=x.y.z make release-src` command, the content of the vote email will be automatically generated in the `/release` directory named `apache-apisix-${x.y.z}-vote-contents`
```
Comment (spacewander) [inline MAINTAIN.md:47]:
```suggestion
   > After executing the `VERSION=x.y.z make release-src` command, the content of the vote email will be automatically generated in the `./release` directory named `apache-apisix-${x.y.z}-vote-contents`
```

File: MAINTAIN.md [hunk 7]
```diff
@@ -42,12 +44,14 @@ via `VERSION=x.y.z make release-src`
...
 4. When the vote is passed, send the [vote result email](https://lists.apache.org/thread/p1m9s116rojlhb91g38cj8646393qkz7) to dev@apisix.apache.org
 5. Move the vote artifact to Apache's apisix repo
 6. Register the release info in https://reporter.apache.org/addrelease.html?apisix
 7. Create a [GitHub release](https://github.com/apache/apisix/releases/tag/2.10.0) from the minor branch
 8. Merge the pull request into master branch
 9. Update [APISIX's website](https://github.com/apache/apisix-website/commit/7bf0ab5a1bbd795e6571c4bb89a6e646115e7ca3)
-10. Update APISIX rpm package
-11. Update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601), and create new branch form master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10.2`
+10. Update APISIX rpm package.
+    > Go to [apisix-build-tools](https://github.com/api7/apisix-build-tools) repository and create a new tag named `apisix-${x.y.z}` to automatically generate rpm package, which can be downloaded from artifacts in action
```
Comment (spacewander) [inline MAINTAIN.md:55]:
```suggestion
    > Go to [apisix-build-tools](https://github.com/api7/apisix-build-tools) repository and create a new tag named `apisix-${x.y.z}` to automatically submit the rpm package to yum repo
```

File: MAINTAIN.md [hunk 8]
```diff
@@ -27,15 +27,16 @@
...
 10. Update [APISIX's website](https://github.com/apache/apisix-website/commit/f9104bdca50015722ab6e3714bbcd2d17e5c5bb3)
 11. Update APISIX rpm package
-    > Go to [apisix-build-tools](https://github.com/api7/apisix-build-tools) repository and create a new tag named `apisix-${x.y.z}` to automatically generate rpm package, which can be downloaded from artifacts in action
-12. Update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601) in [APISIX docker repository](https://github.com/apache/apisix-docker), and create new branch form master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10**.2***`
+    > Go to [apisix-build-tools](https://github.com/api7/apisix-build-tools) repository and create a new tag named `apisix-${x.y.z}` to automatically submit the
+    package to yum repo
+12. Update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601) in [APISIX docker repository](https://github.com/apache/apisix-docker), and create new branch form master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10.2`
```
Comment (spacewander) [inline MAINTAIN.md:39]:
```suggestion
12. Update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601) in [APISIX docker repository](https://github.com/apache/apisix-docker), and create new branch from master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10.2`
```

File: MAINTAIN.md [hunk 9]
```diff
@@ -44,14 +45,14 @@ via `VERSION=x.y.z make release-src`
...
 5. Move the vote artifact to Apache's apisix repo
 6. Register the release info in https://reporter.apache.org/addrelease.html?apisix
 7. Create a [GitHub release](https://github.com/apache/apisix/releases/tag/2.10.0) from the minor branch
 8. Merge the pull request into master branch
 9. Update [APISIX's website](https://github.com/apache/apisix-website/commit/7bf0ab5a1bbd795e6571c4bb89a6e646115e7ca3)
 10. Update APISIX rpm package.
-    > Go to [apisix-build-tools](https://github.com/api7/apisix-build-tools) repository and create a new tag named `apisix-${x.y.z}` to automatically generate rpm package, which can be downloaded from artifacts in action
+    > Go to [apisix-build-tools](https://github.com/api7/apisix-build-tools) repository and create a new tag named `apisix-${x.y.z}` to automatically submit the rpm package to yum repo
 11. Update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601) in [APISIX docker repository](https://github.com/apache/apisix-docker), and create new branch form master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10.2`
```
Comment (spacewander) [inline MAINTAIN.md:57]:
```suggestion
11. Update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601) in [APISIX docker repository](https://github.com/apache/apisix-docker), and create new branch from master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10.2`
```

**Review by spacewander — APPROVED** [review]

**Pull request merged**